sources/NotoSansMono.glyphspackage/UIState.plist

```diff
@@ -0,0 +1,6 @@
+{
+displayStrings = (
+"⌠",
+"∌∉⋢⋣⊄⊃⊅"
```

Comment:
Ha! I thought that I had caused this and fixed it in my sources. I'm sorry if that fix caused you trouble!